.stylelintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  // See https://github.com/wagtail/stylelint-config-wagtail for rules.
+  extends: '@wagtail/stylelint-config-wagtail',
+  rules: {
+    // Would be valuable for strict BEM components but is too hard to enforce with legacy code.
+    'no-descending-specificity': null,
+    // Incompatible with the icon font setup – to be enforced once the icon font is removed.
+    'font-family-no-missing-generic-family-keyword': null,
```

Comment:
Can this ignore rule be scoped go a subset of files only?
This way these issues won't creep into the code while we work on the font removal. 

https://stylelint.io/user-guide/configure/#overrides

Reply:
I’ll switch all legacy icon font usage to a Sass variable, so we can enforce this rule everywhere.